packages/neuron-ui/src/components/SendFieldset/index.tsx

```diff
@@ -45,17 +36,16 @@ const SendFieldset = ({
   isSendMax,
   isMaxBtnDisabled,
   isMaxBtnShow,
-  isAddOneBtnDisabled,
-  isAddBtnShow,
   isRemoveBtnShow,
-  onOutputAdd,
   onOutputRemove,
   onLocktimeClick,
   onSendMaxClick,
   onItemChange,
   isTimeLockable = true,
 }: SendSubformProps) => {
   const [t] = useTranslation()
+  const [showDeleteDialog, setShowDeleteDialog] = useState(false)
```

Comment:
Name it with `is~` to imply it's a boolean, `showDeleteDialog` could be regarded as an action